index.html

```diff
@@ -121,6 +130,115 @@ <h2 class="txt-center">
     $('.js-defaultServerIndexApp').on('click', function(){
       app.loadNewServer("http://localhost:18469/api/v1/");
     });
+
+    var appMenu = Menu.buildFromTemplate([
```

Comment:
The examples on electron's documentation put it in index. It could probably move to js/main.js.